osgp/shared/osgp-dto/src/main/java/org/opensmartgridplatform/dto/valueobjects/smartmetering/SmartMeteringDeviceDto.java

```diff
@@ -36,6 +36,10 @@ public class SmartMeteringDeviceDto implements Serializable {
   private byte[] globalEncryptionUnicastKey;
 
   private byte[] authenticationKey;
+  private byte[] globalEncryptionBroadcastKey;
```

Comment:
Where do you use globalEncryptionBroadcastKey?

Reply:
The unused key is removed.